### PR TITLE
Sim view fix

### PR DESCRIPTION
**Pull request by TristanWright**
fixes an issue where an ec2 cluster log is not available until a refresh or until it's provisioned.

### CODE REVIEW

**Comment by TristanWright** [comment]
@cjh1  PTAL

**Comment by TristanWright** [comment]
@cjh1 PTAL

File: src/redux/actions/taskflows.js [hunk 1]
```diff
@@ -381,6 +391,7 @@ client.onEvent((resp) => {
       }
       case 'cluster': {
         // fetch clusters
+        updateTaskflowMeta();
         dispatch(clusterActions.fetchClusters(null));
```
Comment (cjh1) [inline src/redux/actions/taskflows.js]: Could these two calls overlap?

**Comment by cjh1** [comment]
@TristanWright This fixes the issue for me. One inline comment.

File: src/redux/actions/taskflows.js (hunk 1 above)
Comment (TristanWright) [inline src/redux/actions/taskflows.js]: Yes, I just realized too there's another method named `updateTaskflowMetadata` in the same file so I'l be renaming it too.

**Comment by cjh1** [comment]
LGTM

**Pull request merged**